spk/wallabag/Makefile

```diff
@@ -13,16 +13,14 @@ ADMIN_URL = /wallabag/web/
 RELOAD_UI = yes
 STARTABLE = no
 DISPLAY_NAME = Wallabag
-CHANGELOG = "Initial release"
+CHANGELOG = "Update to 2.3.8, Update to generic service script"
 
 HOMEPAGE = https://www.wallabag.org/
 LICENSE  = MIT
 
 WIZARDS_DIR = src/wizard/
-
-INSTALLER_SCRIPT = src/installer.sh
-SSS_SCRIPT       = src/dsm-control.sh
-
+SERVICE_SETUP = src/service-setup.sh
+STARTABLE = no
 
 INSTALL_DEP_SERVICES = apache-web mysql
 START_DEP_SERVICES = apache-web mysql
```

Comment:
I would not enforce mysql dependency for users expecting to connect to embedded postgresql